lib/nav/collaborator.vue

```diff
@@ -10,7 +10,7 @@
 
 <template>
   <div class="collaborator" @click.stop="selectCollaborator">
-    <avatar class="collaborator-image" :url="user.imageUrl" size="small" ref="avatar"></avatar>
+    <avatar class="collaborator-image" :url="user.imageUrl" size="small" :name="user.name || user.username" ref="avatar"></avatar>
```

Comment:
why is the username in consistent?

Reply:
`username` is what the 3rd party avatar component wants, but it means something different to use because we're _usually_ passing in the person's display name rather than their email